tests/unit/peapods/remote/yamls/encoder.yml

```diff
@@ -0,0 +1,15 @@
+!MyEncoder
+metas:
+  name: myenc  # a customized name
+  workspace: /tmp/blah
```

Comment:
this `tmpdir` should be provided by the `tmpdir` pytest fixture

Reply:
These yaml files are kept just to test the helper function for the right upload. We are not using these yamls to trigger flows/pods.

Reply:
I've removed workspace & made it pymodule simpler